src/app/onecx-permission-remote.module.ts

```diff
@@ -1,27 +1,45 @@
-import { HttpClient } from '@angular/common/http'
-import { NgModule } from '@angular/core'
-import { RouterModule, Routes } from '@angular/router'
+import { HttpClient, HttpClientModule } from '@angular/common/http'
+import { BrowserModule } from '@angular/platform-browser'
+import { APP_INITIALIZER, DoBootstrap, Injector, NgModule } from '@angular/core'
+import { createCustomElement } from '@angular/elements'
+import { Router, RouterModule, Routes } from '@angular/router'
 import { MissingTranslationHandler, TranslateLoader, TranslateModule } from '@ngx-translate/core'
 
 import {
   AppStateService,
   ConfigurationService,
   createTranslateLoader,
+  PortalApiConfiguration,
   PortalCoreModule,
   PortalMissingTranslationHandler
 } from '@onecx/portal-integration-angular'
 import { addInitializeModuleGuard } from '@onecx/angular-integration-interface'
+import { initializeRouter, startsWith } from '@onecx/angular-webcomponents'
+import { AppEntrypointComponent } from './app-entrypoint.component'
+import { AngularAuthModule } from '@onecx/angular-auth'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
+import { environment } from 'src/environments/environment'
+import { Configuration } from './shared/generated'
+
+function apiConfigProvider(configService: ConfigurationService, appStateService: AppStateService) {
+  return new PortalApiConfiguration(Configuration, environment.apiPrefix, configService, appStateService)
+}
```

Comment:
Make sure its not duplicated in SharedModule

Reply:
Ok